neptunecontrib/monitoring/sacred.py

```diff
@@ -100,9 +100,19 @@ def started_event(self, ex_info, command, host_info, start_time, config, meta_in
 
     def completed_event(self, stop_time, result):
         if result:
-            neptune.log_metric('result', result)
+            if isinstance(result, tuple): # logging multiple results
+                for i, r in enumerate(result):
+                    if isinstance(r, float):
+                        neptune.log_metric(f'result_{i}', r)
+                    else: # Ignore non float results # TODO: Find a way of logging objects (log_artifact maybe??)
```

Comment:
Yes, you can log objects either via `log_artifact` on a path to a pickled object or you could use `neptunecontrib.monitoring.utils.pickle_and_send_artifact(obj, 'name.pkl')`

Reply:
Good to know, as some objects I want to store don't actually have a path / pickle associated to them, so this `pickle_and_send_artifact` is a nice wrapper